alb/terraform-examples/nsxt-cloud-data-segment/Create Initial Data Segment.md

```diff
@@ -0,0 +1,192 @@
+# NSXT Cloud Data Initial Segment Creation
+
+Scripts Developed by: William Stoneman</br>
+
+# Introduction
+
+The purpose of this file is to lay out the steps to create the initial NSX-T Data Segment after the NSX-T Cloud Connector has been successfully created. 
+
+NOTE: This is only needed if the initial Data Segment was not specified during the NSX-T CLoud Connector creation.
+
+The information provided below is for demonstration purposes only, and you should update the variables with the information from your environment.
+
+# Low Level Process Review
+
+1. Create tf script:
+
```

Comment:
Can the script content be placed in code blocks with hcl destination?

Example:

```hcl
terraform {}
```